test/JDBC/expected/BABEL-2317.out

```diff
@@ -3,21 +3,16 @@ INSERT INTO t_2317 DEFAULT VALUES;
 INSERT t_2317 DEFAULT VALUES;
 INSERT INTO t_2317 with (dummy_hint) DEFAULT VALUES;
 GO
-~~ROW COUNT: 1~~
-
-~~ROW COUNT: 1~~
+~~ERROR (Code: 33557097)~~
 
-~~ROW COUNT: 1~~
+~~ERROR (Message: syntax error near 'dummy_hint' at line 4 and character position 25)~~
```

Comment:
Let's use a real hint, e.g., tablock, because the purpose of this test is not to check an error. And then you can a separate test case in `BABEL-TABLEHINT` using `dummy_hint`.

Reply:
Done